src/renderEpisodeCard.js

```diff
@@ -0,0 +1,45 @@
+import { renderElementIntoTarget, changeInnerHTML } from './genericFunctions';
+
+const blank = '';
+
+export function renderEpisodeCard() {
+    renderElementIntoTarget({
+        elementId: 'episodeCard',
+        elementType: 'div', 
+        content: blank,
+        elementClass: 'episodeCard', 
+        targetId: 'episodeBox' 
+    });
+    const cardInfo = ['episodeName', 'episodeCodeNumber', 'exibitionDate', 'characterList', 'characters']
+    for (let index = 0; index < cardInfo.length; index++) {
+        renderElementIntoTarget({
+            elementId: cardInfo[index],
+            elementType: 'p', 
+            content: blank,
+            elementClass: cardInfo[index], 
+            targetId: 'episodeCard' 
+        });
+    }
+}
+
+export function setOnclickAttribute(RaMQuery) {
+    const RaMResults = RaMQuery.data.data.episodes.results;
+    RaMResults.forEach(episode => document.getElementById(episode.id).onclick = renderCard.bind(this, RaMResults));
+}
+
+function renderCard(RaMResults) {
+    const RaM = RaMResults[`${event.target.id}`];
```

Comment:
I wouldn't rely on accessing the `event.target` to get the id, if you could simply pass the id on the parameter:
``` js
function renderCard(RaMResults, epidoseId) {
```

Imagine if you moved this function to another file, other developers reading this function would be like "Wtf is this event? where is this coming from?????" and the would need to know this event exists because you are using it in a `dom.onclick` property. 
Remember when I said about a function that used global variables? Also applies here, where your function depends on something that is not being passed explicitly on parameter!. Relying in outer scope means this function can ONLY BE CALLED in a certain way. In your case, this only works when used with `onclick`. But what if you needed to use it in another context? maybe directly after the request, and not on user click?